scripts/grest-helper-scripts/db-scripts/genesis_table.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS grest.genesis;
```

Comment:
Could possibly move this to basics.sql (same as control table)?

Reply:
Definitely an option, I'll think about it